src/script_chainer/config/script_config.py

```
@@ -212,3 +305,69 @@ def update_config(self, config: ScriptConfig) -> None:
         self.script_list[config.idx] = config
         self.init_idx()
         self.save()
+
+    def is_attached_to_prev(self, idx: int) -> bool:
+        """判断第 idx 个脚本是否挂靠到前一个脚本。
+
+        满足以下任一条件即视为挂靠：
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)